nifi-nar-bundles/nifi-framework-bundle/nifi-framework/nifi-framework-core/src/main/java/org/apache/nifi/controller/status/history/ConnectionStatusDescriptor.java

```diff
@@ -61,8 +69,16 @@
         "Queued Count",
         "The number of FlowFiles queued in this Connection",
         Formatter.COUNT,
-        s -> Long.valueOf(s.getQueuedCount()));
+        s -> Long.valueOf(s.getQueuedCount())),
 
+    TIME_TO_BACKPRESSURE(
+        "backpressureEstimate",
+        "Backpressure Estimate",
```

Comment:
You might clarify that the time is displayed in millis. Stating that the default is 6 hours might give the user the impression that the displayed time is also hours.

Reply:
On the graph axis the time is displayed as HH:MM:SS. This link shows an image of what the graph looks like: https://github.com/jmark99/nifi-images/blob/master/count-graph.png
The image is slightly older and the "(count)" specifier is no longer there. If you feel there would still be confusion I can updated the names of the variables to add something like 'millis' or adjust comments, etc. Let me know what you think.

Reply:
I think updating the names of variables would help.

Reply:
After going back and re-running a nifi instance with status analytics enabled I was reminded that the backpressureEstimate variable above is only used as a reference to the graph label. Since the graph does display the time estimate in hours I don't think there should be any confusion.

Reply:
Ah - i wasn't aware of that. No worries from me then.